.github/workflows/ci.yml

```
@@ -82,7 +82,7 @@ jobs:
       - name: Maven Checks
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          $MAVEN clean verify -B --strict-checksums -V -T 1C -DskipTests -P ci
+          $MAVEN clean install verify -B --strict-checksums -V -T 1C -DskipTests -P ci
```

verify is redundant now .. 
```suggestion
          $MAVEN clean install -B --strict-checksums -V -T 1C -DskipTests -P ci
```